third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2121,6 +2121,63 @@ def groupby(
         used to group large amounts of data and compute operations on these
         groups.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame({'Animal': ['Falcon', 'Falcon',
+            ...                                'Parrot', 'Parrot'],
+            ...                     'Max Speed': [380., 370., 24., 26.]})
+            >>> df
+               Animal  Max Speed
+            0  Falcon      380.0
+            1  Falcon      370.0
+            2  Parrot       24.0
+            3  Parrot       26.0
+            <BLANKLINE>
+            [4 rows x 2 columns]
+
+            >>> df.groupby(['Animal'])['Max Speed'].mean()
+            Animal
+            Falcon    375.0
+            Parrot     25.0
+            Name: Max Speed, dtype: Float64
+
+        We can also choose to include NA in group keys or not by setting `dropna`
+        parameter, the default setting is `True`:
+
```

Comment:
we can omit this repeat

Reply:
I deleted `the default setting is True:` sentence, but I guess it's better to have keep this sentence for showing `dropna` and `as_index` cases. Pandas doc does it doo: https://screenshot.googleplex.com/AdoicovZxRoQZV2

Reply:
Oh, I didn't mean this (probably the github highlighting was not accurate). I meant the following lines, as they already appear at lines 2126-2127.

            >>> import bigframes.pandas as bpd
            >>> bpd.options.display.progress_bar = None

Reply:
Thanks. Done 👍